nncf/parameters.py

```diff
@@ -96,6 +96,23 @@ class CompressWeightsMode(StrEnum):
     E2M1 = "e2m1"
 
 
+@api(canonical_alias="nncf.BackupPrecision")
+class BackupPrecision(StrEnum):
+    """
+    Defines a backup precision for weight compression.
+    :param FP: Stands for original floating-point precision of the model weights (either FP16 or FP32).
```

Comment:
Are models with BF16 weights considered?

Reply:
yes, the weights should remain in the original BF16 precision

Reply:
Should I add `fp16/bf16/fp32` to `Statistics of the bitwidth distribution` string instead of `fp16/fp32`?

Reply:
Just mention BF16 in the doc strings along with mentioning fp32 and fp16